Content.Shared/Jukebox/JukeboxListPrototype.cs

```diff
@@ -0,0 +1,19 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Jukebox;
+
+[Prototype("musicList")]
+public sealed class MusicListPrototype : IPrototype
+{
+    [IdDataField] public string ID { get; } = string.Empty;
+    [DataField("songs")] public MusicListDefinition[] Songs { get; private set; } = Array.Empty<MusicListDefinition>();
+}
+
+[DataDefinition]
+public sealed partial class MusicListDefinition
```

Comment:
i dont think this is really necessary? name can perfectly be a field in the component, path needs to be of `ResPath` type, copyright is never used and cant you get the song lenght directly from the file or something?